client/dashboard/emails/add-professional-email/index.tsx

```diff
@@ -182,7 +186,20 @@ const AddProfessionalEmail = () => {
 
 	return (
 		<PageLayout
-			header={ <PageHeader /> }
+			header={
+				<PageHeader
+					prefix={
+						<RouterLinkButton
+							className="add-forwarder__back-button"
+							icon={ arrowLeft }
+							iconSize={ 12 }
+							to={ emailsRoute.to }
+						>
+							<Text variant="muted">{ __( 'Emails' ) }</Text>
+						</RouterLinkButton>
+					}
```

Comment:
Can you move this to a reusable component so we don't need to replicate it on each step?